supported_parts.md

```diff
@@ -149,6 +149,7 @@
 - LM75
 - ADPD1080
 - MAX11205
+- MAX14001
```

Comment:
Please add MAX14002 as well

Reply:
Hi Travis, I've already added the max14002 as supported parts.

Thank you